src/OgAccess.php

```diff
@@ -112,18 +111,24 @@ public static function userAccess(EntityInterface $group, $operation, AccountInt
     // To reduce the number of SQL queries, we cache the user's permissions
     // in a static variable.
     if (!$pre_alter_cache) {
-      $permissions = array();
-
-      foreach (Og::getUserMemberships($user) as $membership) {
+      $permissions = [];
+      $user_is_group_admin = FALSE;
 
+      if ($membership = Og::getUserMembership($user, $group)) {
```

Comment:
`isMember` only returns a bool though? We need the membership below to check the roles on it.